src/modules/announcements/announcement.controller.ts

```diff
@@ -26,6 +27,9 @@ export class AnnouncementController {
 
   // add a Announcement
   @Post()
+  @ApiOperation({
+    summary: 'add a Announcement',
+  })
```

Comment:
Add summary in response doc folder

Reply:
yeah i wanted to add that but summary key does not exist in Api Response options object